bestbook/views/__init__.py

```diff
@@ -153,7 +153,7 @@ def post(self):
         for elem in data["observations"]:
             key, value = list(elem.items())[0]
             if key in all_observations:
-                all_observations[key] = f"{all_observations[key]}{self.MULTI_CHOICE_DELIMITER}{value}"
```

Comment:
Yes.  In the original code, `'key'` would still need to be in single quotes...
`all_observations[key] = f"{all_observations['key']}{self.MULTI_CHOICE_DELIMITER}{value}"`